pegasus/sites.v3/code.org/views/theme_common_head_before.haml

```diff
@@ -9,7 +9,7 @@
 %link{rel:'shortcut icon', href:'/images/favicon.ico'}
 %link{rel:'apple-touch-icon', href:'/images/apple-touch-icon-precomposed.png'}
 
-%script{src:'https://cdn.optimizely.com/js/12977480133.js'}
+%script{src:'www.googleoptimize.com/optimize.js?id=OPT-KBX3C3L'}
```

Comment:
Missed adding the `https://` here so Optimize installation was failing—fixed this in Dropbox on 5/24